chronicle-front/src/app/components/searchbar/searchbar.component.html

```diff
@@ -0,0 +1,27 @@
+<mat-form-field class="example-chip-list">
+    <mat-label>Search tags</mat-label>
+    <mat-chip-list #chipList aria-label="Fruit selection">
+      <mat-chip
+        *ngFor="let fruit of fruits"
```

Comment:
Fruits? Let's change this to something more appropriate for this application.

Reply:
We updated